sdcm/logcollector.py

```diff
@@ -281,6 +281,31 @@ def get_grafana_annotations(self, node):  # pylint: disable=inconsistent-return-
             LOGGER.warning("unable to get grafana annotations [%s]", str(ex))
             return ""
 
+    @staticmethod
+    def dashboard_exists(node: CollectingNode, uid: str) -> bool:
+        """Check on Grafana server, that dashboard exists
+
+        Send request to Grafana and validate that dashboard with uid
+        provided as "uid" parameter is available and could be requested
+
+        Arguments:
```

Comment:
as for docs, my personal preference is the reST/sphinx format and not the google one
https://stackabuse.com/python-docstrings/

but having some would always help

Reply:
we haven't defined the "official" docstring format, let's discuss it on our QA meeting. Let's leave as is.

Reply:
leave as is